compiler/rustc_macros/src/newtype.rs

```diff
@@ -137,6 +148,45 @@ impl Parse for Newtype {
             quote! {}
         };
 
+        let default_derives = if ord {
```

Comment:
You should be able to just append to the `derives.push` to add `PartialOrd` and `Ord`, instead of duplicating the list of derives below.

Reply:
What type would I need (and how would I get the values for `Ord` and `PartialOrd` in order to push onto `derives`?

Reply:
Do you mean `derives` or `derive_paths`?